sw/device/tests/otbn_isa_test.c

```diff
@@ -0,0 +1,144 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/entropy_testutils.h"
+#include "sw/device/lib/testing/otbn_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+/**
+ * This test runs every instruction in OTBN's ISA and checks the result.
+ *
+ * It reuses the otbn's `smoke_test.s`,
```

Comment:
nit: Coming to this without DV context, I had to do a little digging to figure out where `smoke_test.s` originally came from. Maybe:
```suggestion
 * It reuses the OTBN simulator's `smoke_test.s`,
```